README.md

```diff
@@ -24,9 +24,10 @@ A curated list of awesome SCION tools, applications, libraries and resources.
 - [eXpress SCION Router (XSR)](https://github.com/netsys-lab/express-scion-router) - A SCION border router making use of XDP and P4. :wrench: :construction:
 
 ## Applications
-- [caddy-scion](https://github.com/scionproto-contrib/caddy-scion) - SCION enabled HTTP webserver :wrench:
+- [SCION Application docs](https://docs.scion.org/projects/scion-applications/en/latest/) Compilation of technical information for supported SCION applications.
+- [HTTP proxy](https://github.com/scionproto-contrib/http-proxy) - SCION enabled HTTP Forward/Reverse proxy :wrench: (docs[https://scion-http-proxy.readthedocs.io/en/latest/index.html])
```

Comment:
(docs[https://scion-http-proxy.readthedocs.io/en/latest/index.html])

I suspect you meant:

[docs](https://scion-http-proxy.readthedocs.io/en/latest/index.html)

Reply:
may be be all of it within parenthesis.